doc/build.md

```diff
@@ -100,16 +100,14 @@ troubleshooting, etc.
 
 ## Docker
 
-        # Build using all available cores
-        docker build -t grin .
-
-        # run in foreground
-        docker run -it -v grin:/usr/src/grin grin
-
-        # or in background
-        docker run -it -d -v grin:/usr/src/grin grin
```

Comment:
Why remove the foreground/background part?

Reply:
Sorry, just merged as you were commenting...

Reply:
No problem, still think it's interesting to have a up to date instructions on how to start the image.